mobile/src/bundles/common/components/input/input.tsx

```diff
@@ -52,16 +59,35 @@ const Input = <T extends FieldValues>({
                     <Text category={TextCategory.INPUT}>{marker}</Text>
                 </View>
             )}
+
+            {iconName && (
+                <View
+                    style={[
+                        globalStyles.alignSelfCenter,
+                        globalStyles.pv15,
+                        globalStyles.pl5,
+                        styles.iconContainer,
+                    ]}
+                >
+                    <Icon
+                        name={iconName}
+                        size={iconSize}
+                        color={Color.PRIMARY}
+                    />
+                </View>
+            )}
+
             <TextInput
                 onChangeText={onChange}
                 value={value}
                 onBlur={onBlur}
                 {...props}
                 style={[
                     globalStyles.flex1,
-                    globalStyles.pl10,
                     globalStyles.Input,
-                    !marker && globalStyles.borderRadius5,
+                    !marker && !iconName && globalStyles.borderRadius5,
+                    !iconName && globalStyles.pl10,
+                    !!iconName && styles.inputBorder,
```

Comment:
```suggestion
                    !!iconName ? styles.inputBorder : globalStyles.pl10,
```

Reply:
In this case we get two eslint errors (Unexpected negated condition and Redundant double negation). So, I think it's better to leave as it was